### PR TITLE
Adds storybook

### PR DESCRIPTION
**Pull request by hstove**
Over the weekend, I wanted to setup Storybook. I've never used it, and I thought it would be cool to play around with different `@blockstack/ui` components. I think it'll be helpful as we build out a set of reusable components both in the app and in our design system.

One thing I couldn't figure out was how to get automatic React prop definitions from Typescript into Storybook. I've set it up exactly as the docs say, and I know that `react-docgen-typescript-loader` is picking up our components, but they're not being displayed correctly. Oh well, maybe we'll figure it out later, and I think it's still useful enough to have.

To use:

~~~bash
yarn storybook
~~~

### CODE REVIEW

**Comment by kyranjamie** [comment]
+1 for storybook, super helpful when building components

Wasn't able to build branch though:

`"blockstack-keychain": "file:../keychain",` should this [point to this repo](https://github.com/blockstack/blockstack-keychain)? 🤔

**Comment by hstove** [comment]
yep - i'm actually fixing this right now. But if you want to try it ASAP, you'll have to clone and build the `blockstack-keychain` folder as a neighboring folder.

(FYI, I'm renaming and publishing it as `@blockstack/keychain`, which is part of what I'm doing right now)

**Comment by hstove** [comment]
@kyranjamie , that issue is resolved :+1:

Also FYI - I didn't actually put much of anything into our Storybook, I just wanted to set it up to play with. I would like to see us include a bunch of our Waffle components in there.

File: .storybook/addons.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import '@storybook/addon-actions/register';
+import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
+import 'storybook-addon-jsx/register';
```
Comment (kyranjamie) [inline .storybook/addons.js:4]: See the no new line icon. Would be good to use same `.editorconfig` as blockstack js. Currently there is inconsistent spacing in this repo, tabs & spaces

**Comment by kyranjamie** [comment]
One other thought: Unless you're using this as just a test playground, shouldn't this code be in the ui repo itself?

**Comment by aulneau** [comment]
I think we should close this, this probably should live in @blockstack/ui? cc @hstove @kyranjamie

**Pull request closed without merging**